pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/AggregateOperatorTest.java

```diff
@@ -79,7 +77,8 @@ public void shouldHandleUpstreamErrorBlocks() {
     DataSchema inSchema = new DataSchema(new String[]{"group", "arg"}, new ColumnDataType[]{INT, INT});
     DataSchema outSchema = new DataSchema(new String[]{"sum"}, new ColumnDataType[]{DOUBLE});
     AggregateOperator operator =
-        new AggregateOperator(OperatorTestUtil.getDefaultContext(), _input, outSchema, calls, group, inSchema);
+        new AggregateOperator(OperatorTestUtil.getDefaultContext(), _input, outSchema, inSchema, calls, group,
+            false, true, false, false);
```

Comment:
TODO: many of these functions are setup to run in both leaf and intermediate, we need to mix and match test modes as well

Reply:
Is there a way to execute a pre-defined plan, which we can use that to auto-gen the mix-match the test cases?